### PR TITLE
[DOC] Advertise nilearn's in built globbing

### LINKED ISSUE #810: Globbing should be advertised in doc/examples.

### PR DESCRIPTION
**Pull request by man-shu**
- Closes #810

Changes proposed in this pull request:

- add a section in [this](https://nilearn.github.io/stable/auto_examples/00_tutorials/plot_nilearn_101.html) example about globbing

### CODE REVIEW

**Comment by man-shu** [comment]
Just for the record, I ended up using different example and dataset than what I previously mentioned in the related issue #810

File: examples/00_tutorials/plot_nilearn_101.py [hunk 1]
```diff
@@ -59,17 +59,49 @@
...
+    Path(datasets.get_data_dirs()[0]) / "haxby2001" / "subj*" / "anat*"
+)
+
+# %%
+# Now we can smooth all the anatomical images at once
+anats_all_subjects_smooth = image.smooth_img(anats_all_subjects, fwhm=5)
+
+# %%
+# This is a 4D image containing one volume per subject
+anats_all_subjects_smooth.shape
```
Comment (bthirion) [inline examples/00_tutorials/plot_nilearn_101.py:92]:
```suggestion
print(anats_all_subjects_smooth.shape)
```

**Review by bthirion — COMMENTED** [review]
Looks great ! Thx

**Review by Remi-Gau — APPROVED** [review]
good with me!!!

thanks

**Pull request merged**